runhouse/main.py

```diff
@@ -98,6 +102,116 @@ def ssh(cluster_name: str, up: bool = typer.Option(False, help="Start the cluste
     c.ssh()
 
 
+def _print_status(config):
+    """
+    Prints the status of the cluster to the console
+    :param config: cluster's  config
+    :return: cluster's  config
+    """
+    envs = config["envs"]
+    config.pop("envs", [])
+
+    # print headlines
+    daemon_headline_txt = (
+        "\N{smiling face with horns} Runhouse Daemon is running \N{Runner}"
+    )
+    console.print(daemon_headline_txt, style="bold royal_blue1")
+    console.print(f"{config['name']}", style="bold underline")
```

Comment:
not sure if this is possible, but is there a way to hyperlink the cluster rns address that we display? 

for example, if the cluster name is: /jlewitt1/rh-cpu-https its corresponding Den URL would be:
https://www.run.house/resources/jlewitt1:rh-cpu-https